docs/tutorial/examples.md

```diff
@@ -50,7 +50,7 @@ guide!).
 
 You can find the full list of "How to?" in the sidebar. If there is
 something that you would like to do that is not documented, please join
-our [Discord server][] and let us know!
+our [Discord server][discord] and let us know!
 
-[discord server]: https://discord.com/invite/electron
+[discord]: https://discord.com/invite/APGC3k5yaH
```

Comment:
```suggestion
[discord]: https://discord.com/invite/electronjs
```

Sorry, super nitpicky, but I think we use `https://discord.com/invite/electronjs` - just think we should keep that in case this one expires 🙂

Reply:
@VerteDinde, I actually had that one originally for this PR, but then noticed that all the other ones in our docs use `https://discord.com/invite/APGC3k5yaH`, so I changed it to be consistent.

Meanwhile on the `electron/governance` repo we use `https://discord.gg/electronjs` in our playbook responses. 🙂 I can go either way, not sure if there's any advantage to one or the other.

I don't know the history to `https://discord.com/invite/APGC3k5yaH` - if it's just a historical artifact then I can change this PR to update all of the links to be `https://discord.com/invite/electronjs`.

Reply:
Oh nice, sorry! I just checked and it looks like `https://discord.gg/electronjs` does work - let's use that to keep it short and be consistent 🙇‍♀️ I think `APGC3k5yaH` is a relic from when we first opened the Discord and couldn't use custom URLs for invites, but it's been a while.